ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/string/LPadTest.java

```diff
@@ -32,10 +33,21 @@ public void shouldPadInputString() {
 
   @Test
   public void shouldPadInputBytes() {
-    final ByteBuffer result = udf.lpad(input(), 7, padding());
+    final ByteBuffer result = udf.lpad(BYTES_123, 7, BYTES_45);
     assertThat(result, is(ByteBuffer.wrap(new byte[]{4,5,4,5,1,2,3})));
   }
 
+  @Test
+  public void shouldAppendPartialPaddingString() {
+    final String result = udf.lpad("foo", 4, "Bar");
+    assertThat(result, is("Bfoo"));
+  }
+
+  @Test public void shouldAppendPartialPaddingBytes() {
+    final ByteBuffer result = udf.lpad(BYTES_123, 4, BYTES_45);
+    assertThat(BytesUtils.getByteArray(result), is(new byte[]{4,1,2,3}));
+  }
+
```

Comment:
Original author had added this test on rpad but not rpad, just noticed and added them here.